go.mod

```diff
@@ -289,6 +289,7 @@ require (
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51 // indirect
 	github.com/klauspost/compress v1.17.7 // indirect
 	github.com/knadh/koanf v1.4.1 // indirect
+	github.com/kr/fs v0.1.0 // indirect
```

Comment:
Please run `make modtidy-all` in this repo

Reply:
I run but it is already indirect dependency because `github.com/pkg/sftp` package imports it.